### PR TITLE
aquainfra ogc processes update

### PR DESCRIPTION
**Pull request by MarkusKonk**

### CODE REVIEW

File: tools/aquainfra_ogc_api_processes/aquainfra_ogc_api_processes.xml [hunk 1]
```diff
@@ -85,10 +86,45 @@
...
+                    <option value="OLCI_S3B">OLCI_S3B: [ 400 412 443 490 510 560 620 665 674 681 709 754 779 866 ]</option>
+                    <option value="OLI">OLI: [ 443 482 561 655 865 ]</option>
+                    <option value="SeaWiFS">SeaWiFS: [ 412 443 490 510 555 670 865 ]</option>
+                    <option value="VIIRS_JPSS1">VIIRS_JPSS1: [ 411 445 489 556 667 746 868 ]</option>
+                    <option value="VIIRS_JPSS2">VIIRS_JPSS2: [ 411 445 488 555 671 747 868 ]</option>
+                    <option value="VIIRS_SNPP">VIIRS_SNPP: [ 410 443 486 551 671 745 862 ]</option>
+                </param>
+				<param name="output_option" label="Output option" optional="false" type="select">
+                    <option value="1">1: for standard output with five variables</option>
+                    <option value="2">2: for extensive output including memberships of all water types</option>
```
Comment (bgruening) [inline tools/aquainfra_ogc_api_processes/aquainfra_ogc_api_processes.xml:125]:
```suggestion
                </param>
                <param name="output_option" label="Output option" optional="false" type="select">
                    <option value="1">1: for standard output with five variables</option>
                    <option value="2">2: for extensive output including memberships of all water types</option>
```

File: tools/aquainfra_ogc_api_processes/aquainfra_ogc_api_processes.xml [hunk 2]
```diff
@@ -85,10 +86,45 @@
...
 			</when>
+            <when value="owt-classification">
+				<param name="input_data_url" label="Input data" optional="false" help="URL (stored in a .txt file) to your input file." type="data" format="txt"/>
+				<param name="input_option" label="Type of input" optional="false" help="If type of input is csv, the output is a text file (.csv). If it is sat, the output is a NetCDF file (.nc) of the same dimensions and geo-reference as the input file." type="select">
+                    <option value="csv">csv: for text data input (first line wavelength, following lines remote-sensing reflectance)</option>
+                    <option value="sat">sat: for satellite product input containing reflectance (e.g., Sentinel-3 OLCI Level-2)</option>
+                </param>
+				<param name="sensor" label="Sensor name" optional="false" help="Spectral band configuration of satellite mission (includes adaptation to sensor spectral response functions)." type="select">
```
Comment (bgruening) [inline tools/aquainfra_ogc_api_processes/aquainfra_ogc_api_processes.xml:99]: the indentation is off here, how can this happen in an automatic generated script?

Comment (MarkusKonk) [reply tools/aquainfra_ogc_api_processes/aquainfra_ogc_api_processes.xml:99]: Well, I need to admit that this update was not as automated as planned :). 
I always forget about the indentation. I think I need to revise my local linter...

Comment (bgruening) [reply tools/aquainfra_ogc_api_processes/aquainfra_ogc_api_processes.xml:99]: Use the force Luke. The Galaxy Language server!

Comment (MarkusKonk) [reply tools/aquainfra_ogc_api_processes/aquainfra_ogc_api_processes.xml:99]: Oh, that is a good one!

**Comment by MarkusKonk** [comment]
@bgruening I don't get why the linting fails. It did not fail before and doesn't fail locally.

**Comment by bgruening** [comment]
https://github.com/galaxyecology/tools-ecology/actions/runs/11479238977/job/31945153652?pr=136#step:7:3334

You miss some  metadata in your shed.yml

**Comment by MarkusKonk** [comment]
It looks like this is referring to the xamlstarlet tool. I have the necessary metadata in the shed https://github.com/AquaINFRA/tools-ecology/blob/master/tools/aquainfra_ogc_api_processes/.shed.yml

**Comment by MarkusKonk** [comment]
And it looks it's running the linting and tests for all tools. I am a bit confused....

**Pull request closed without merging**